### PR TITLE
feat: show user profile on dashboard with ability to edit

### PR DESCRIPTION
**Pull request by nickreynolds**
- editing/setting email and avatar from dashboard is functional, but not implemented as designed (since design is not ready yet)
- display of email address is not as designed (since designs not updated to reflect this field yet)

### CODE REVIEW

File: packages/components/src/TCRUserDashboard/DashboardStyledComponents.tsx [hunk 1]
```diff
@@ -78,7 +78,95 @@ export const StyledDashboardActivityDescription = styled.div<StyledDashboardActi
 `;
 
 export const StyledUserInfo = styled.div`
-  width: 277px;
+  width: 333px;
+  ${mediaQueries.MOBILE} {
```
Comment (sruddy) [inline packages/components/src/TCRUserDashboard/DashboardStyledComponents.tsx]: FYI `mediaQueries.MOBILE` is `max-width: 1030px`, `mediaQueries.MOBILE_SMALL` is `max-width: 640px` not sure you ant the larger size here. We should name these better when we make updates to style guide.

File: packages/components/src/TCRUserDashboard/DashboardStyledComponents.tsx [hunk 2]
```diff
@@ -78,7 +78,95 @@ export const StyledDashboardActivityDescription = styled.div<StyledDashboardActi
...
+    width: 100%;
+  }
+`;
+
+export const StyledUserProfile = styled.div`
+  width: 333px;
+  ${mediaQueries.MOBILE} {
```
Comment (sruddy) [inline packages/components/src/TCRUserDashboard/DashboardStyledComponents.tsx]: Same here

File: packages/components/src/TCRUserDashboard/DashboardStyledComponents.tsx (hunk 1 above)
Comment (nickreynolds) [reply packages/components/src/TCRUserDashboard/DashboardStyledComponents.tsx]: `MOBILE` is the point where we change the layout of the page (stack components on top  of each other), so I think it's correct here

**Comment by dankins** [comment]
:tada: This PR is included in version 4.9.3 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**